recipes/cafe/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "4.2.1" %}
+{% set sha256 = "f58da8f79eda33c233f0bddaa0f780eaa7fdb3b874d8841f0a7f2cba3cc3533f" %}
+
+package:
+  name: cafe
+  version: {{ version }}
+
+source:
+  url: https://github.com/hahnlab/CAFE/releases/download/v{{ version }}/CAFE-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
+
+test:
+  commands:
+    - which cafe
```

Comment:
can you add a test like `cafe --help` or --version?

Reply:
sure, a test on the version has been added.